spec.html

```diff
@@ -13887,15 +13798,13 @@ <h1>
       <emu-clause id="sec-stringgetownproperty" type="abstract operation">
         <h1>
           StringGetOwnProperty (
-            _S_: unknown,
-            _P_: unknown,
+            _S_: an Object that has a [[StringData]] internal slot,
+            _P_: a property key,
```

Comment:
we should maybe `dfn` this, though I guess that's can happen outside of this PR.

Reply:
We should definitely dfn that :-)